src/http/types.ts

```diff
@@ -897,13 +897,23 @@ export const DeviceBasicLockStatusProperty: PropertyMetadataNumeric = {
     writeable: false,
     type: "number",
     states: {
-        1: "1",         //TODO: Finish naming of states
-        2: "2",
-        3: "Unlocked",
-        4: "Locked",
-        5: "Mechanical anomaly",
-        6: "6",
-        7: "7",
+        1: "MANUAL_UNLOCK",         //TODO: Finish naming of states
+        2: "AUTO_UNLOCK",
+        3: "PW_UNLOCK",
+        4: "FINGERPRINT_UNLOCK",
+        5: "APP_UNLOCK",
+        6: "MANUAL_LOCK",
+        7: "KEYPAD_LOCK",
+        8: "APP_LOCK",
+        9: "AUTO_LOCK",
+        10: "LOW_POWER",
+        11: "VERY_LOW_POWER",
+        12: "MULTIPLE_ERRORS",
+        13: "LOCK_OFFLINE",
+        14: "MECHANICAL_ANOMALY",
+        15: "VIOLENT_DESTRUCTION",
+        16: "PW_LOCK",
+        17: "FINGER_LOCK",
```

Comment:
Please revert this change

Reply:
After you reverted this part i will merge, please don't change anything after that.